### ISSUE TITLE
Exception Thrown

### ISSUE THREAD

**Issue by MaroisConsulting** [Hefaistos68/VSLauncherX#27]
When I start the app, I get 

"System.InvalidOperationException: 'A valid active Window is needed to update the Taskbar.'"

in Line 452 MainDialog.SetupTaskbarTasks()

**Comment by Hefaistos68** [comment]
Whats the Windows version you are running under?

**Comment by MaroisConsulting** [comment]
> Whats the Windows version you are running under?

Windows 11 Pro 22621.3447

**Comment by Hefaistos68** [comment]
Right, don't have W11 currently available, but will look into this when I get the time. Should be an easy fix.

**Comment by Hefaistos68** [comment]
Someone with W11 please verify so we can close the issue.
